terragrunt/aws/network/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "vpc_id" {
```

Comment:
We'll need a few more outputs for the Lambda function:
- `aws_security_group.api.id`
- `module.url_shortener_vpc.private_subnet_ids`

[Here's an example of how we'll then use them](https://github.com/cds-snc/scan-files/blob/233cca8d3f400d35616d76b71479ddf0238d7554/terragrunt/aws/api/lambda.tf#L13-L14) when we create the function.

Reply:
Done. Hopefully I have done them correctly.